<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buywindow</class>
 <widget class="QWidget" name="buywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>购入</string>
  </property>
  <widget class="QPushButton" name="buy_back_btn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>476</width>
     <height>297</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QLabel" name="buy_total1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>     商品名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>单个总价</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="buy_lineEdit4"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="buy_total0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="buy_total3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="buy_num2"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="buy_price4"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="buy_price2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="buy_lineEdit0"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="buy_num1"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="buy_lineEdit2"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="buy_total2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="buy_price3"/>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="buy_num3"/>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="buy_num0">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="buy_lineEdit1"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="buy_lineEdit3"/>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="buy_price0"/>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="buy_price1"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>数量</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="buy_total4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="buy_num4"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>价格</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buy_total_btn">
        <property name="text">
         <string>结算</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="buy_total">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="buy_btn">
      <property name="text">
       <string>购入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>tips:先点击结算再购入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buy_back_btn</sender>
   <signal>clicked()</signal>
   <receiver>buywindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
